tutornotes/plugin.py

```diff
@@ -103,4 +116,15 @@ def _mount_edx_notes_api(volumes, name):
         for key, value in config.get("unique", {}).items()
     ]
 )
-tutor_hooks.Filters.CONFIG_OVERRIDES.add_items(list(config.get("overrides", {}).items()))
+tutor_hooks.Filters.CONFIG_OVERRIDES.add_items(
+    list(config.get("overrides", {}).items())
+)
+
+# Notes public hosts
+@tutor_hooks.Filters.APP_PUBLIC_HOSTS.add()
+def _notes_public_hosts(hosts: list[str], context_name: t.Literal["local", "dev"]) -> list[str]:
+    if context_name == "dev":
+        hosts += ["notes.{{ LMS_HOST }}:8120"]
```

Comment:
This should be `{{ NOTES_HOST }}:8120` (and same below)

Reply:
That's a nice catch. Fixed